### ISSUE TITLE
error in PASTEC

### ISSUE THREAD

**Issue by adriludwig** [JBerthelier/PiRATE#53]
Hi Jeremy,

I'm not being able to run PASTEC in the last days.  After 40 hours running it gave an "Fatal error : exit code 1()". I checked the format of input and it is right with short names and 60 bp width

The last lines of log are:
START LaunchPASTEC
step 1 : insert banks in DB
- profiles bank:
/home/jeremy/galaxy/tools/Pipeline/PASTEC/WORK/Pfam27.0_GypsyDB_PiRATE_hmmer.hmm
- repbaseNT bank: /home/jeremy/galaxy/tools/Pipeline/PASTEC/WORK/PiRATE_nucl.fa
- repbaseAA bank: /home/jeremy/galaxy/tools/Pipeline/PASTEC/WORK/PiRATE_prot.fa
END LaunchPASTEC
2020-08-25 11:40:47 - PASTEClassifier - INFO - Split fasta file
save the 8405 input sequences into 85 batches
2020-08-25 11:40:47 - PASTEClassifier - INFO - Launch PASTEC on each batch
submitting job(s) with groupid 'genome_PASTEC' (2020-08-25 11:40:47)
waiting for 85 job(s) with groupid 'genome_PASTEC' (2020-08-25 11:40:48)
job 'PASTEC_77' in error, re-submitting (1)
job 'PASTEC_77' in error, re-submitting (2)

Any clues on what's going on?
Thanks!

**Comment by JBerthelier** [comment]
Dear adriludwig, 

I suggest you to not run all you input sequences at once, but to do several runs of PASTEC by separating your sequences. 
This can help to overlap this problem. 

Let me know if this helped, 

Best, 

Jeremy

**Comment by adriludwig** [comment]
Thanks. I tryied even with only 10 sequences and it is not working. 
Now error is:
Fatal error: Exit code 1 () [blastall] WARNING: Database is empty
[blastall] WARNING: Database is empty Traceback (most recent call
last): File "/home/jeremy/Pipeline/REPET_2.5/bin/PASTEClassifier.py",
line 405, in <module> iPASTEClassifier.run()

**Comment by JBerthelier** [comment]
Dear adriludwig, 

Sorry for my late responds, 
Unfortunatly, I never saw this error before.
It sould work perferctly with only 10 sequences.
Could you copy paste some sequences, I could have a look on it and maybe find the reason.

Otherwise, you can also contact the administartor of PASTEC: https://urgi.versailles.inra.fr/Tools/PASTEClassifier/PASTEClassifier-tuto

Best regards

Jeremy

**Comment by adriludwig** [comment]
That's really weird. Yesterday, i run another file, with more and distinct sequences and it worked. So I went back to the 10 sequences to test it again and... error! It just don't make sense, but maybe it is something simple that I'm not figuring out. 
Here are them:

That's really weird. Yesterday, i run another file, with more and distinct sequences and it worked. So I went back to the 10 sequences to test it again and... error! It just don't make sense, but maybe it is something simple that I'm not figuring out. 
Here are them:
That's really weird. Yesterday, i run another file, with more and distinct sequences and it worked. So I went back to the 10 sequences to test it again and... error! It just doesn't make sense, but maybe it is something simple that I'm not figuring out. 

Thanks a lot for your attention, Jeremy!

Here are them:
file:///home/jeremy/Téléchargements/Galaxy864-[FASTA_Width_on_data_860].fasta

**Comment by adriludwig** [comment]
>genome1
ATTGAGTGTTGTAAAAGTATTTTGCTTCTTATTGTTCTATCATGAAAGCCTCCAGTTGAT
GTCTCATCTAACGGTGTCACGCATCACGTGTCTGGAAGAGCGGTAAGGCAACGCTCTGGC
AGGTATCTCCGAATGCGGTTTACGGTCACACATGTTTCTATCTTTATGGGTTGAATATAT
CATTATATATAATAATATAGAATACCATCCTACTATAAATATGTACTATCCTTCCATTCA
AAGTATAAGTTCAGTAACTCGATTGTATTTCATGCAATGCCAATGTTTTCGATGTGCGCT
GATTATCACAACGATCATACTGTGTTCGCCAAAACATGAGAGAACCTCGAAGCAAAGTGT
GCCCCTTCATCCGCACAACTCGCTGGGAGTTCGCTGACAAAACGTGTTTGCAAGAGAAAC
AAGGCGACTTCAACGAAATCATCTCATGCTTAGGCCTGAGTTTCTTGAAGGATCATATAG
GATTCTCGGATCCCATGTCTTCCCAAGGCGTCACAGGTTTGTTGATATTCCACTTGAACG
TTCCCTATTGTGGGGGGTTTCAGCAGATCGCGAAATACTGGAAGCAATCCAGTTTTATGA
AAGCACTGATCGAGCTGTGACTGACTGCGGTACCAGTAACTGCTCATGCCAGCAATCATC
GTACCAAACAAGCAAACGTGCGACCTCAGCCTGAATATCCTTTCTGCAGATCTGATGAAA
GGCCACCTCTTCGTGGAAAACCACCAGTTCACGTCTCATTTCGCTGGTGTATTGCATTGG
TCTGTTCACAGTGTAAGTAGATTGACGTGGAAGCAATTATCGAGGTCAGCGCATTCCCGG
ATATATCGAGGTTTTCCATCCTTTCAGGAAGGTCACCAAATTTGTGTCGGCCCTTTGACA
AATTTGATCATCTGTCTTTTATTTGCTTATAGTCATTCGCAATGTTTACATCATTGGTTA
TTCCATGTGACGACGGCATTGGTAGATTCGAGGAACCAAAAATTTACAGCAAAGCCTTCT
GGAGATTCTCACATTCGGACTCGATGAAGTCGAAGCATCCGGTATTCGTAAATCGACTGT
TTGTTCCTCTCAATCAAGAAGTGGCCATTAAATATTTTAAAAGAGTAGTTTGCGCCGTGT
TTTTTCGAAGAGCTCATGAAAGGAGATAACCCAATTCTGTTCTGTAAACCTTTTTCCAAT
GTTTTCGATGTGCGCACTTGTTTTCGCCTTTGCTGACATTTCAAATATTCCGCCTGGCGA
CATAAACTTTCTTCAGCTGCAATAGCAAACCACTTTCATGGACCGTTTTTATTCACCGAA
AAGTAAAGTAAAAGCGAAGACCAAGAGTACAAAGGAACATAAAGTTGAGCGCGCGGATTA
TTTAAGAAATGCCCCTTTGCGTGACTATTGCGAGTGGAAGAGTGTGAGGTGCGTCGAAGG
GAAAGTGGTTCGAGTAGACCACATAGATCGTCACGATAAAGTGTTGGATATTCACTTGTT
GCCACCGACCGTGGGAGATATCCATTTAACAAGCTGCTCGCTGGACTACGCACTTCATAC
GCGAGCGTTGCCACGTACATTGAAGTACTGCCGCATTAACCGCAATCGGCTTCGTGGGTC
TATCGGCTTACGAACCCTTCCGGAACACCTCGTACATCTTAATCTCTCTACGAATCAGCT
GGTTGGTCCTGTCGATCTTACGGAATTACCACGCAAGCTTATAACCCATCGATCTGTCGG
ACAACCGTATTCAGCAATCAGTTATTTTCTTTGGCCAGCTTCCCCCAAACTTGATATACG
TCAAGTTTTACTTTGTTGCGGGAAATAACCTAATCGGCGAGCTCCGTAGCACATCCACGG
AAAATGTGGAAAGGCTTGGAGAAATTTTCCCGGGCATTTCGTTGAAGCATATTCATATAG
AATGAGAAATTTCTGCCCGGTTAATCCACACGCTTTAGTTGAAAAGAGTTTACTATTCTG
TAGTAATACTTCTATTTTGCATTCCGCGCTTTGGCTGCAAATCTTGCGTGTTTGAGTAGT
AAATATAAGCGACCATATTCATGAAGAAAACGTAGGGAATGTACGCGCCAGAACAATCCA
TTGGGCTATTCTTCTACGTAGCGGATTCCCACCTTTCTAAGTCAATTTACACCAGACGAT
GTATAAACACAATAGCAACGAAATCCGTACACGGATGCTGAGAGGTTGCACACCTTATCC
CGCGCTACTGGTAAACCGCATGCCTTCTCGTGCACTGTCGTTTTATTTAAGCTATAAGAT
TAATAACTTGATGGTATTTCGTATAACCTCAACCAGCCACCGGATTCATGATCATCGTCC
TTGTTGGCCTTAGACGCTTCATCCACAGAGTATTCGCATAATAATTTTCCGCTGTGTTCT
ATTTCAACCGCCATCGGC
>genome2
AGTACACAGGTCATGTAGCGCTCTTAGAACGCACCAATCTAATATCAGCCTTACACGGGC
ATTCGGCATTTAACTTGCCTTTGGAAAGCCGTGATGTACAAACAAGAGTTGTGTTTGCGC
TTCCCCTCCGCTTCGGGTCAGCCAGTAATTTGTGATAATGTCTCACAAGTAATATATGGA
TAGAATCACGGGCGTGCGTATTCGTTATGTAACATTTCTCGCACTAGACTACACATCTTT
CGAAACTATGAATGGACCGCAGACATTGATGGACACTCGGTGCATCAACTGTCGGATGCT
AACCGAGGATGTTCACTCCAGCATCCGAGAATACATGCACCGAGTGTCCAGCAATGTCTT
CCGAGGATGCATAGATTGGAAAGCTTTATCGTCAAGTACGAGCAGTGGCTAACGATTAGG
GAAGCCCGTGATGCTATCCGTGTCCTCTGTGTGAGACATGGTCACAAATTACTGGCTGAC
CCGAAGCGGAAGGGAAGCACAAACACAACTCTTGTTTGTACATCACGGCTTTCCAAAGGC
AAGTTAAATGCCGAATGCCCGTGTAAGGCCGAGATTAGAGTGGTGCGTTCGAAGAGCGCC
ACAGGACCTGTGTACTACCGCATCTCCCGTAGCTCTGTTTTTGAGCACACGCAGTGCGCC
ATAATATTCCCGAAGGCTTCAGCCAAAGGAATCATCCAGCTCGACAGCTTTCAGCAGTAC
GCAGAAAGTACCCCTGATGCGACTGGACGGCAAATGCAGAAATTCATGCAGA
>genome3
TAAATGTGCAGAAGTCAGTAACGCTCTTATAGCAGCTACTCCGATCACAATTACCGCTAG
CCTATTACCATTGAAACTGCATCTTGATTGCATTGGTATAGTAAGTATGCAAGACCGTTT
GCGCCTTATCCCTACCGTTATTTCTTTACCCCGTTCTACCTTTCTTTCTCACCGTTCTGT
CGAGTACATTTTTCGGAGGTAAAATGACCCCCCCCCCCAGGGAGCCAAGTCTGCGGGACT
CGAAGTGGCGACCACGGTTCCGTACATGAACCCGCTACTCGCTTGCCTACCTAAGCTAGT
CGTTACGTTCGCTAAAATCAAGGGGAGACCTCGTTTGAGCTAGCAGCGGTGCGATGATGA
AGTTGCTTATCAATCGAACAGGATGCAACCTCAAAATGTACTATTGAACTGTGGTAAACT
GTTTGCTCTTACAAATGTGAGAAAGCATTTTTATTACGACATTAAGTAATATGCGATAAG
AAGAAAGTATTGCTCATAACGTCTTTTTCTTATATGGTACCATGGCTTCCGAAATAATTA
CTGTCAAGTTCTACATACTATAAGATGAAAGCATAATTATATCTTCAGCTGTAATCCATA
GGTAATCTTGTTGCGGGAAGCTAAAATAGTTCTCGATGATGATGTAATTTTATTTTCTGT
AGCCGCAAAGAACTACAGCTATAGCTTCTTTTAGATACGTGTCACCGATGGGATTATATA
TGATGCACATAAAAATCCGCGCCTTTTCCTTTACTAACTAACGTTTATAAGTCACAATTT
AAAAGTAAAAGCATATAATAGCTAGAAACGTCACAGCAGG
>genome4
CCGAATACGAAAAGCTCCATCAAAGTCTGCTGTGGAAGCGTCGATCGGTCGACTCTGCCG
ATACCGTCATCACATGCAATGAAAAAGCGCTGCGAACATTTGGATTTGCATAAGATATTA
ATTGAAACTGCGAGAAGCACAGGAAATGAATGAACATTGGGTGAGAACGAGCTGTCCATC
ATACTTCATTGACTCAACATGCTTTCCACTCCTGTTATCATGCACCATCCCAGTCTATAT
ACCTCAACATAAGACATCAAAAATACCTCTCGATTCGTGTTCCTTGATGAGGTGCCCGTC
TACAGCGCACATTGGTACGGGTACGCTCATACAGGTGGTCACGGAGATCTCGACGTCAGG
GCGTCACTTCTATAAGGCACATGCGGCAATGTACGTCCACCAGCCGATATAATCACTCGA
TGAAGCAGGACTTTAAGCATGCATGTGAATGCGAATGCCTACCATGTAATGCTGATGGGG
AAGTCGAGGGTTCTTGTTCGTTATGTAAAAACGAGGATGGAAGATGCACACCCAGCTTGC
AGTTTCTTTCATGGCTCCGTGCTTCGCCGCTTTCGCCTCGGCTGAATTTGTGTAGAAACG
TTCAACGATAATCGTAAGGATCAATACTACTCAGGAAATTGCCAGGTCTATGCGAAGACA
CTCTTGAGTAGTCACATTGACCATGTTGTTTTGCATGGTACCCTTGATTCTGTAATGATC
ATTGTCAAGTTCTATATTCCATAAGATGAGAGCTTACTTAGATGCTTCCCTTTAATTGAG
AAGTAATATTGATGCGGGGAACTCAAAAGTTATTAACACACATGTTGCATGAATTTAACA
TATTTTCGTACCCACAAAGAACTACAGTTAAAGCTTCTCTCAGCATTCGTGTTAGAACTG
CCATTTCAAATGATGCACATGAAATCCGCACCTGTTCCTTTACTAACTAACCAACCAGAT
GTCCTAGTCATAAGCTATATAGGCAAAATTGCATATATTAATAGCATCGAAATGCGTCAC
AATTTCAGGCTAAGTATATAATGTCTTGTACAATGTGGCTTTCAATAGGTTATACTCCAA
ATGCATGTACAAAGAAGCTAGCACACTGGAGTTGCAATTTGGCAGGATATACAATAGTAA
GCCTATTATAGCAAATAGTAGCCTGATGTGACGCTTCGAGCTATTATAAGATTTTGCCTA
TATGTTATGACTAGGACATCTGGTTGTTAGTTAGTAAAGGAAAAGTCCGGATTTCATGTG
CATCATTTGAAATGC
>genome5
GAGTAGCCTGGGAACGGGTTATTACAAGGCAGCTGGGACAACTGGATTGGAGGTTCTTGC
ATGGAGTATAAGACGACTTGACATAATGTTCCTCAGCCATAACTGGGCGTTCGACGCGTT
GTAAACGGTGTCTTTTCTCAAATACAATCAGCTGAGGGAATGTGCCTGACGGCATGAGGT
TTTCATGATATGAATAATACAGGTCAAACTACTCGCTTAGTAAAACAGTTAATGGTGCCA
CATGGCCTTAGTGGGTTCTTCGGAGAATATATTCCGAACTTACAATCGAACGCGTTTTTT
CCATAAATGTCAAATTGTAATTTCTAGAAAAAACGCGTTCGACTGTAAGTTCGTTATTTA
TTCACCAAAAAACACCACTAAGGCCATGTGGCACCCTTACTGTTCCACTAAGCGCGTTAT
TTTTAACGTCATAAAAACGTCATACTTTCAGGCAGATTCCCTAAGTCGATTGTGCCTGAG
AAAAGACACCGATTACAACTCGTCGAACCGACAAGTTGCGGATGGACAATTTGGCGAGTC
GTCTGATAGTCCAAGCAAGAACCTTCAAGGGTGTTGTCCCAGCTGCCTTCTAGATAAAGA
GTATCTAAGGCTGGTCGGAAGGGAGGTCAAACATACTGCACCACTCAGCTGATTAGCAGA
TAGCGTTAAAAATCTGAGACCTTTCGGGAGCTTTTTGAGGGAGATTTCTCCAGTGAATTT
GTTTTGAGTGAGATTGAGGGCTTGAACGTCAGCGGGAAGACCGTCTAGGTCGAGACTACC
AGTCAGCTTGTTGCTAGCAAGAGATATTACCTCCATCTTCTCCGGGAGCCCAGCAAGTTG
GATTGTACCACTGAGGGCGTTCATGAACATCCGCAGCGTTTTCATACT
>genome6
GCGTTCTGTCCTTCTCTCTTTCCCTTCTTTTATCATTTTGGCCATCTCTATTTAGCTCGT
TTACCTACTAAAAGTTGACATTTTTCCGAAATAAGTGGCGAAAATTTGACGCGAAATAAG
TGCACCCGTCTGAGAGCCATGGCAGGCGTCAACAGGAATTGCAGCGGAAAGCGGTTTTTG
TTTCCAATCCAATACACGCACAGCGAGATGCGACGTGGACTGGAGGGTTTCACGATGAAA
AACACGGCGCAAAATACTGTTTGAAACACTTCATGGTCATTTCTTGAGACAGCCAGATTC
GAAATACGCACGCGTACTATACGCAGCCAGATTCGAGTTTTTCAATAACTCTCAAATTGT
ATTTTGCTATCATTGCGACACAATGTTGTGAAACCGGTAACTGCTCATACAAATTTGCGC
TGTGTTGTATTCAGTTGCTATTTCGAATCCTCCGTTTATTGTAAATCTTTCTTTATGCAA
TACGATGTTTTCAAAAGTGTACGCGCCACCGAATTCATGACTATCGTCCTTGTTAACCTT
CGAGGCTTCATCCCAAAGTATCCGCATACTAAAATTCCGGTGTTTTTATTCAACTTTCCT
AGCGTTAATTTTGAATATTTCTGTATGAACAAAATGTTGTCAAAAGTGTACGTGGGAGTT
TTCTTCGCATGCGCCGACGACGGAATCGGCAGGTTCGACCGATCCATGCTGTCTCAGCAA
AGCTTAATGGAGCTTTTCTTATTTGGGCTCGCGAAACCAGAGAAGATATGTCGCCATCGG
GATGATCCGGCTGATGTATGTGAATGGAAGGAGGTGAAATGTAACGGTGATGGGGAAGTC
GAGGAGTTTCGGTGGGTTTTTAAACAAGAGAATGGAACTGGTACAGTTGGCTTCGAGTTT
CTTCCATGCAGTATAAAAACACTGA
>genome7
AAGTGTACTGTACCAGCAAAGACGTTGAAACCCAAGTCCAGTATTTGGAGATTCATCGGC
AGGGTTCTTAAGTCAATATTACCGTGGAGTCCGCATCGAACGAGCTTGCAAAATCGAAGC
GCCCGCGGAAGCTTCCGGGTGCAGAGTTCCGTGTAGTGACTGTAGTGTATTCTATTTATG
ATAACATTCATTGTGGTGCCAGGCATCCAGTGAAAGTCAACGCGGCGGTAGGAGTGTATT
CGCCAAGAAATACCCAAAATCGTCAGACTGTCGCTGCACCCGACGCCACGCCACTTGCAA
ACGTGAGTTGGCTCTTCCTCGTCGCTCTTGAGAAGATGGATTGGATGGATTGCATCTGTC
AGTTGCGTTCTGTCAGAGACACCGATGAGCCGATACATACAATACACCTGGATGTCGTCT
CCAGGTGTGGTGTCAATCGCGAGGGCGAATGCCCAATGCAGTATGGTGATGGCCATTTTA
GAATTCATTTAAAAGCAGAAAATTTAGCTACAATTTGAAAATGTGAAAGGTTTGAGCTTC
GTAGAAAAGGGGCGATGCGTTTCCACTAGATGGTGCGCCGCAGTGTGAACATTGTGTCAC
AATACGGTGGGCATGTCCGTTTAGAGTAGAATTCATTGCATATATAATACTTCAAAGGGC
ATTGTGTTTAAACTTGAGCTTTTCAAAAGAGTAAGGTCACTTGATAAAGCATATTTCACC
GTTAAACTCAACATTATATTCGAACGACTTTTCACCAAAGCAAAAAAA
>genome8
TATATGGATATGCGTCACGATTACTTACTAAATTTATAGTGTTTCATAAAATGTGGCTTT
CTTATAGCTAATACTCCTAATGCATGTACGAAGAAGCAAGCACAGTGGAGTTACAATTCT
TATAATCGTATGGCTATTATAGAAAATATTGTACCCAGTGTTCAGCCTGCTGTGACGTTT
CTAGCTATTATATGCTTTTACTTTTAAATTGTGACTTATAAACGTTAGTTAGTAAAGGAA
AAGGCGCGGATTTTTATGTGCATCATATATAATCCCATCGGTGACACGTATCTAAAAGAA
GCTATAGCTGTAGTTCTTTGCGGCTACAGAAAATAAAATTACATCATCATCGAGAACTAT
TTTAGCTTCCCGCAACAAGATTACCTATGGATTACAGCTGAAGATATAATTATGCTTTCA
TCTTATAGTATGTAGAACTTGACAGTAATTATTTCGGAAGCCATGGTACCATATAAGAAA
AAGACGCTATGAGCAATACTTTCTTCTTATCGCATATTACTTAATGTCGTAATAAAAATG
CTTTCTCACATTTGTAAGAGCAAACAGTTTACCACAGTTCAATAGTACATTTTGAGGTTG
CATCCTGTTCGATTGATAAGCAACTTCATCATCGCACCGCTGCTAGCTCAAACGAGGTCT
CCCCTTGATTTTAGCGAACGTAACGACTAGCTTAGGTAGGCAAGCGAGTAGCGGGTTCAT
GTACGGAACCGTGGTCGCCACTTCGAGTCCCGCAGACTTGGCTCCCTGGGGAGGGGGGTC
ATTTTACCTCCGAAAAATGTACTCGACAGAACGGTGAGAAAGAAAGGTAGGGATAAGGCG
CAAAAGGTCTTGCATATTGTCCC
>genome9
GTGCCGCTGGATAGAATAGGTAGATAGGCTCGAAAGGACATAACGCCTATGACTTAAGCT
GGGTCCAGTGAGTGGTCCAAGAAGCTATGGTAATGGCAATTGCCCATAGCAGGGATATAG
TAGAAAATCCTTCTGTAGCTATGTAACCTGCCCTTAATGGGATGTTATGGTAAATAGCTT
GCATCCTCAAGAAGAGTTGAAAAAAATACAGATGTTCCTACCTGAATAACGAGAAGACGT
CAGTGGGCGAACCTCTCCGCAATACGATCCTTGCATGTGGGAAGCCTGTGAAGCTGGAAC
GGCATACTAGGTTTCAGGAATCCTACGCCATCCTTCACGAACCTCACACATCAGCATGAG
ATGATATCCTTGAAGTCCCCTCGTTTGTCTTGCAAACACGTTTTCTCCTTGAATTCCCCA
TCGTGTACCCACACAAGTGGCTATCAGGTTTGTGCCAAGTGGCTGAGCGAGGAGAAAAGT
TGTTGGAAACATTCTCATCTTCTACCTTATCTGTTGAATGCCAAATACACCCTCCACTGT
CGATAATCCCTATCTGTGGACGGCCCGAATTGAATTCGTAAATACGATTCCGCACAGTGA
AAACATTCGAAACAAGTTGTACCAATCCTGGATGTACCATAGCAGTCTATACCAACTGGT
GTTGAGGTCTCGACTGACGGAAATACCATGCCCTGCATATCAAGTTCCACTACATAAAGA
CGTTTTTGACGTATACAGAATACGTTGTTGACCAAGCGCTCTTCGGCCTGTCCGTTTTCC
GGGATGGGAAGCCCTTCCGATTCGGTAAGCGCACTTTGGTTTACTACACATCCTGCACAG
GGTTTTCATTGATACATCCCAGTAACAGATATCTGAGAGACCAGCCCTGCCAGTACGGCC
CTCCCAGACACGTTATGCGCTGCTTTTCTTGCCACAGGCGATTTTGACACTTTTAAATAG
AAGAATAACTTTC
>genome10
GGTTCGTGAAGGATGGCGTAGGATTCCTGAAACCTAGTATGCCGTTCCAGCTTCACAGGC
TTCCCACATGCAAGGATCGTATTGCGGAGAGGTTCGCCCACTGACGTCTTCTCGTTATTC
AGGTAGGAACATCTGTATTTTTTTCAACTCTTCTTGAGGATGCAAGCTATTTACCATAAC
ATCCCATTAAGGGCAGGTTACATAGCTACAGAAGGATTTTCTACTATATCCCTGCTATGG
GCAATTGCCATTACCATAGCTTCTTGGACCACTCACTGGACCCAGCTTAAGTCATAGGCG
TTATGTCCTTTCGAGCCTATCTACCTATTCTATCCAGCGGCACAGGCTTGCGAGTACCAT
GTTGAGGGGAGGTTGAACCATTGGCATCTCCTCATTACTCAGGGGTGAACACTCGAAAGA
TCCACTAAATAAAATTGCGCTCTGTTTATTTCAACTCTTCTAGCGTCATTTTTGAATGTT
TCAGCCAGTTCCGAGAAAGATTGCTCTGAGTTGAGGTGAATCGTTGCATTTCACAAGGCA
TTCTAACAATTTCGAAGTTGTGGACGTTAAATGTAATATTTTGTGCGGCAAAAAAGGCTA
TCAGCTGCCGGAGAGGCTGAACTATCCGGGTGCTTCGAACGATAGGCCGAATGGAACAGT
ACGGGTGCCGTCTGGAGTTTCCTGTTGGGGTGTGTGGTGTCAGAGCCACCGCCTGGTTGA
CGAAACCAAGGACCTCACAGTGCAATATATAAGGAAAGATTGACTTCGAGAAAAATCCAG
AGTTAACATTCCTGTGGTCCATTATATGACGGGTTCAACTGTACGTTT

**Comment by JBerthelier** [comment]
Dear adriludwig

I have run PASTEC with your inputs and I got this error

2020-09-14 03:33:44 - PASTEClassifier - INFO - START PASTEClassifier parallelized
2020-09-14 03:33:44 - PASTEClassifier - INFO - Fasta file name: genome.fa
2020-09-14 03:33:44 - PASTEClassifier - INFO - Running STEP 1 of PASTEClassifier parallelized: DetectTEFeatures
START DetectTEFeatures_parallelized
Fasta file name: genome.fa
split the sequences in batches Homogeneous in size 20kbp
launch the programs on each batch
prepare bank 'PiRATE_nucl.fa'...
prepare bank 'PiRATE_prot.fa'...
prepare bank 'Pfam27.0_GypsyDB_PiRATE_hmmer.hmm'...
submitting job(s) with groupid 'genome_detectTEFeaturesParallelized' (2020-09-14 03:37:43)
waiting for 1 job(s) with groupid 'genome_detectTEFeaturesParallelized' (2020-09-14 03:37:43)
execution time per job: n=1 mean=79.572 var=0.000 sd=0.000 min=79.572 med=79.572 max=79.572
collect the results and load them into tables
collect results 'TR'
collect results 'polyA'
collect results 'SSR'
collect results 'ORF'
collect results 'Profiles'
collect results 'TE_BLRn'
collect results 'TE_BLRtx'
collect results 'TE_BLRx'
table 'genome_TR_set' exists (36 lines)
table 'genome_SSR_set' exists (29 lines)
table 'genome_polyA_set' exists (0 lines)
table 'genome_ORF_map' exists (100 lines)
table 'genome_Profiles_path' exists (6 lines)
table 'genome_TE_BLRn_path' exists (0 lines)
table 'genome_TE_BLRtx_path' exists (0 lines)
table 'genome_TE_BLRx_path' exists (0 lines)
table 'genome_HG_BLRn_path' doesn't exist
table 'genome_rDNA_BLRn_path' doesn't exist
table 'genome_tRNA_map' doesn't exist
END DetectTEFeatures_parallelized
2020-09-14 03:39:19 - PASTEClassifier - INFO - Finished STEP 1 of PASTEClassifier parallelized: DetectTEFeatures
2020-09-14 03:39:19 - PASTEClassifier - INFO - Running STEP 2 of PASTEClassifier parallelized: Classification
START LaunchPASTEC
step 1 : insert banks in DB
- profiles bank: /home/jeremy/galaxy/tools/Pipeline/PASTEC/WORK/Pfam27.0_GypsyDB_PiRATE_hmmer.hmm
- repbaseNT bank: /home/jeremy/galaxy/tools/Pipeline/PASTEC/WORK/PiRATE_nucl.fa
- repbaseAA bank: /home/jeremy/galaxy/tools/Pipeline/PASTEC/WORK/PiRATE_prot.fa
END LaunchPASTEC
2020-09-14 03:41:38 - PASTEClassifier - INFO - Split fasta file
save the 10 input sequences into 1 batches
2020-09-14 03:41:38 - PASTEClassifier - INFO - Launch PASTEC on each batch
submitting job(s) with groupid 'genome_PASTEC' (2020-09-14 03:41:38)
waiting for 1 job(s) with groupid 'genome_PASTEC' (2020-09-14 03:41:38)
execution time per job: n=1 mean=4.040 var=0.000 sd=0.000 min=4.040 med=4.040 max=4.040
2020-09-14 03:41:48 - PASTEClassifier - INFO - Started post processing of Classification
2020-09-14 03:41:48 - PASTEClassifier - INFO - Reverse complement
2020-09-14 03:41:48 - PASTEClassifier - INFO - Rename headers according to Wicker's code
2020-09-14 03:41:48 - RenameHeaderClassif - INFO - START RenameHeaderClassif
2020-09-14 03:41:48 - RenameHeaderClassif - INFO - No json classif file provided, skipping
2020-09-14 03:41:48 - RenameHeaderClassif - INFO - END RenameHeaderClassif
2020-09-14 03:41:48 - PASTEClassifier - INFO - Adding noCat classification based on bestHit
2020-09-14 03:41:48 - NoCatBestHitClassifier - INFO - START NoCatBestHitClassifier
2020-09-14 03:41:48 - NoCatBestHitClassifier - INFO - Started analyzing input classification file
2020-09-14 03:41:48 - NoCatBestHitClassifier - INFO - Number of Ok: 0, Number of NoCats 10, Number of Chimeric: 0 , Total: 10
2020-09-14 03:41:48 - NoCatBestHitClassifier - INFO - Finished analyzing input classification file
2020-09-14 03:41:48 - NoCatBestHitClassifier - INFO - Started fetching sequences by pre-selected headers
2020-09-14 03:41:48 - NoCatBestHitClassifier - INFO - Finished fetching sequences by pre-selected headers
2020-09-14 03:41:48 - NoCatBestHitClassifier - INFO - Started aligning noCats with valid classifications
Beginning Blaster (version 2.25)
Mon Sep 14 03:41:48 CEST 2020
Exception catch:  program blastn return with error value: 65280, stopping blaster
Exit with value 1
Mon Sep 14 03:41:48 CEST 2020

It seems that all these sequences are clasified as noCat and that PASTEC has issue to go further and produce the input.

**Comment by adriludwig** [comment]
Thanks very much for your time and help.
Actually, initially, I had a file with almost 1000 sequences and, as I got the error, I tried to split into two files (the error continued) and then also tried with the first 10 sequences. From what you said today, I  believed the searches generated distinct errors because I know in the initial file there are TEs that should be classified. So, I just went back to the 1000 sequences file and took a look to try to figure out if there were some "weird" sequences and actually I found some with large regions os missing data. So, I deleted all "N" and tried again. It worked while the original file keep giving error! So, I guess those regions were creating some problems for PASTEC. So, that's it... thanks for helping and sorry for not having figured this out earlier.
